src/app/resources/video.py

```diff
@@ -16,6 +17,31 @@ def post(self):
 
         return response.json(), response.status_code
 
+    @login_requerido
+    def get(self):
+        offset = int(request.args.get('offset', 0))
+        cantidad = int(request.args.get('cantidad', 10))
+        params = {'offset': offset, 'cantidad': cantidad}
+        response = media_server_api.get_videos(params)
+        if response.status_code != 200:
+            return response.json(), response.status_code
+
+        # remover los videos del usuario actual
+        videos = response.json()
+        videos = list(filter(lambda video: (video['usuario_id'] != g.usuario_actual), videos))
+
+        response = self._obtener_autores(videos, offset, cantidad)
+        if response.status_code != 200:
+            return response.json(), response.status_code
+
+        autores = response.json()
+        for i, video in enumerate(videos):
+            # TODO ver si se puede mejorar: obtener el autor del video de la lista de autores
+            autor = [autor for autor in autores if autor['id'] == video['usuario_id']][0]
```

Comment:
me perdi que es el 0 del final, tambien poner una constante con nombre

Reply:
el 0 es por que por que el resultado del for está generando una lista con 1 elemento, quería hacer sin la lista pero no me lo tomaba, por eso está el comentario puede que python tenga una mejor solución para ésto.

Reply:
ah ok, no creo que esta bien asi. Se podria hacer con un filter tambien sino, pero fue